### PR TITLE
Add some export fields

### PR DESCRIPTION
**Pull request by hlohaus**
### 1. Why is this change necessary?

https://issues.shopware.com/issues/SW-24885

Now there are tests for the export! 💃 

### 6. Checklist

- [x] I have written tests and verified that they fail without my change
- [x] I have squashed any insignificant commits
- [x] This change has comments for package types, values, functions, and non-obvious lines of code
- [x] I have read the contribution requirements and fulfil them.

### CODE REVIEW

File: tests/phpunit.xml.dist [hunk 1]
```diff
@@ -12,8 +12,11 @@
          verbose="true">
 
     <testsuites>
+        <testsuite name="Shopware_ExportTests">
+            <!-- Must run before the other tests, because they create new products.-->
+            <file>./Functional/Core/sExportTest.php</file>
+        </testsuite>
         <testsuite name="Shopware_AllTests">
-            <directory>./Unit</directory>
```
Comment (mitelg) [inline tests/phpunit.xml.dist]: why did you remove the unit tests?

File: tests/Functional/Core/sExportTest.php [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+        $this->connection->exec($sql);
+
+        $fileName = $this->generateFeed(99);
+        static::assertFileEquals(
+            __DIR__ . '/fixtures/productexport/' . $fileName,
+            $this->cacheDir . $fileName
+        );
+    }
+
+    private function generateFeed($feedId)
```
Comment (mitelg) [inline tests/Functional/Core/sExportTest.php]: typehints and return type

File: tests/Functional/Core/sExportTest.php [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+ * trademark license. Therefore any rights, title and interest in
+ * our trademarks remain entirely with us.
+ */
+
+use Shopware\Models\ProductFeed\ProductFeed;
+use Shopware\Models\ProductFeed\Repository;
+use Shopware\Tests\Functional\Traits\DatabaseTransactionBehaviour;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
```
Comment (mitelg) [inline tests/Functional/Core/sExportTest.php]: doubled header

File: engine/Shopware/Core/sExport.php [hunk 4]
```diff
@@ -881,7 +881,14 @@ public function sCreateSql()
                 d.shippingtime,
                 d.shippingfree,
                 a.topseller,
+                a.pseudosales,
                 a.keywords,
+                a.metaTitle,
```
Comment (shyim) [inline engine/Shopware/Core/sExport.php:887]: Did you checked metaTitle with translations?

File: tests/phpunit.xml.dist (hunk 1 above)
Comment (hlohaus) [reply tests/phpunit.xml.dist]: Otherwise it will be executed twice.

File: engine/Shopware/Core/sExport.php (hunk 4 above)
Comment (hlohaus) [reply engine/Shopware/Core/sExport.php:887]: Good point

File: tests/phpunit.xml.dist [hunk 5]
```diff
@@ -12,8 +12,11 @@
          verbose="true">
 
     <testsuites>
+        <testsuite name="Shopware_ExportTests">
```
Comment (shyim) [inline tests/phpunit.xml.dist]: Can you please create your own data set in your test like in the [BenchmarkBundle](https://github.com/shopware/shopware/blob/d27c42c6c3056024bdf9fa52822450b057ab91f0/tests/Functional/Bundle/BenchmarkBundle/Providers/assets/products_images.sql) tests instead of creating a new test suite

**Comment by shyim** [comment]
Thanks for your Contribution! :blue_heart:
